LICENSE

```diff
@@ -1,6 +1,6 @@
-The MIT License (MIT)
+MIT License
 
-Copyright (c) 2014 Katrina Owen, _@kytrinyx.com
+Copyright (c) 2017 Exercism, Inc
```

Comment:
I have taken this directly from https://choosealicense.com/licenses/mit/ which has both the word "copyright" and the "(c)".